src/Integration.UnitTests/Binding/BindCommandTests.cs

```diff
@@ -105,14 +108,14 @@ public void BindCommand_CanExecuteBind()
             // Act + Verify
             Assert.IsFalse(testSubject.WpfCommand.CanExecute(projectVM), "No connection");
 
-            // Case 4: Connecting
+            // Case 4: busy
             this.sonarQubeService.SetConnection(new Uri("http://127.0.0.0"));
-            testSubject.Controller.IsConnecting = true;
+            testSubject.Controller.State.IsBusy = true;
```

Comment:
Two things (repeated in other places)
1. The state extraction
2. IsConnecting + IsBinding were combined into a general IsBusy flag that simplifies things.